e2e/e2e_test.go

```diff
@@ -25,7 +25,6 @@ import (
 	_ "github.com/hashicorp/nomad/e2e/quotas"
 	_ "github.com/hashicorp/nomad/e2e/scalingpolicies"
 	_ "github.com/hashicorp/nomad/e2e/scheduler_sysbatch"
-	_ "github.com/hashicorp/nomad/e2e/scheduler_system"
```

Comment:
Note the comment in the group below: we import all the tests anyways so that we get a compile check on commit.